index.html

```diff
@@ -0,0 +1,164 @@
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Healthy Food</title>
+  </head>
+  <body>
+    <div class="container-fluid">
+      <div class="row">
+        <nav
+          class="navbar navbar-inverse text-white socialMediaIcons"
+          id="home"
+        >
+          <div class="icon-bar navbar-right">
+            <a href="#" class="facebook"><i class="fa fa-facebook"></i></a>
+            <a href="#" class="twitter"><i class="fa fa-twitter"></i></a>
+            <a href="#">My page</a>
+            <a href="#">Logout</a>
+          </div>
+        </nav>
+      </div>
+      <div class="row">
+        <nav class="navbar" id="navbar">
+          <div class="navbar-header">
+            <a class="navbar-brand" href="#">
+              <img src="./images/img4.png" alt="navbar logo imag" />
+            </a>
+          </div>
+          <ul class="nav navbar-nav navItem text-secondary" id="navMenu">
+            <li class="active"><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#program">Program</a></li>
+            <li><a href="#sports">Sports</a></li>
+            <li><a href="#sponsers">Sponser</a></li>
+            <li><a href="#">Nutrition</a></li>
+          </ul>
+
+          <div class="humgurger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+      </div>
+      <main class="main1">
+        <div class="row introduction">
+          <p>"Welcome to Healthline Nutrition!"</p>
+          <h1 class="secondHeader">
+            The Definitive Guide<br />
+            to Healthy Eating in Real Life
+          </h1>
+
+          <p id="second-p">
+            Healthy eating doesn’t have to be complicated. It’s entirely
+            possible to nourish your body while enjoying the foods you love.
+          </p>
+          <h3>2022.10.15(SAT) ~ 16(SUN)</h3>
+          <h5>@ Do not focus on how much you eat ,Focus on what you eat .</h5>
+        </div>
+        <div class="row mainProgram" id="program">
+          <h3>Main Program</h3>
+          <hr />
+          <div class="col-sm card">
+            <img src="./images/fruits.jpg" alt=" fruits" />
+            <h4 class="card-title">Fruits</h4>
+            <p class="card-text">
+              Fresh, frozen, or canned fruits are great choices. Try fruits
+              beyond apples and bananas such as mango, pineapple or kiwi fruit
+            </p>
+          </div>
+          <div class="col-sm card">
+            <img src="./images/vegtables.jpg" alt=" vegtables" />
+            <h4 class="card-title">Vegtables</h4>
+            <p class="card-text">
+              Fresh, frozen, or canned fruits are great choices. Try fruits
+              beyond apples and bananas such as mango, pineapple or kiwi fruit
+            </p>
+          </div>
+          <div class="col-sm card">
+            <img
```

Comment:
- [ ] `optional`
- [ ] please make these cards aligned like the template provided
![image](https://user-images.githubusercontent.com/66230306/198056416-34a06edf-44f7-4066-96ef-4841d33734c0.png)